data-exports/deploy/data-exports-aggregation.yaml

```diff
@@ -477,6 +498,118 @@ Resources:
                   - !Sub "arn:${AWS::Partition}:s3:::${ResourcePrefix}-${DestinationAccountId}-data-exports/*/${AWS::AccountId}/*"
                   - !If [NonEmptySecondaryDestinationBucket, !Sub "arn:${AWS::Partition}:s3:::${SecondaryDestinationBucket}/*/${AWS::AccountId}/*", !Ref 'AWS::NoValue']
 
+  BucketWriteBlockerLambdaRole:
+    Type: AWS::IAM::Role
+    Condition: DeployWriteBlocker
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service: lambda.amazonaws.com
+            Action: sts:AssumeRole
+      ManagedPolicyArns:
+        - arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
+      Policies:
+        - PolicyName: BucketPolicyAccess
+          PolicyDocument:
+            Version: '2012-10-17'
+            Statement:
+              - Effect: Allow
+                Action:
+                  - s3:GetBucketPolicy
+                  - s3:PutBucketPolicy
+                Resource: !GetAtt DestinationS3.Arn
+
+  BucketWriteBlockerPolicyLambda:
+    Type: AWS::Lambda::Function
+    Condition: DeployWriteBlocker
+    Properties:
+      Runtime: python3.12
+      Handler: index.lambda_handler
+      FunctionName: !Sub ${ResourcePrefix}-CID-WriteBlocker
+      Role: !GetAtt BucketWriteBlockerLambdaRole.Arn
+      Code:
+        ZipFile: |
+          import os
+          import json
+          import boto3
+
+          POLICY_SID = os.environ['POLICY_SID']
```

Comment:
Where is this policy defined/deployed?

Reply:
line 561 reference 'AllowReplicationWrite'
https://github.com/aws-solutions-library-samples/cloud-intelligence-dashboards-data-collection/blob/0a92b0ccadfbe5f3a9112cbfb1cfc15d951295ac/data-exports/deploy/data-exports-aggregation.yaml#L561

this one is hardcoded reference to this SID :
https://github.com/aws-solutions-library-samples/cloud-intelligence-dashboards-data-collection/blob/0a92b0ccadfbe5f3a9112cbfb1cfc15d951295ac/data-exports/deploy/data-exports-aggregation.yaml#L290